src/components/Avatar/Avatar.component.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import styled from 'styled-components';
+
+function Avatar() {
+  return (
+    <>
```

Comment:
When you are rendering just a single node element, the React fragment is not necessary.